.mkchlog.yml

```diff
@@ -0,0 +1,23 @@
+# mkchlog was introduced after this commit
+skip-commits-up-to: 1eb6e0c7d85fbc5e3f265de069b91be2cb722359
+
+sections:
+    security:
+        title: Security
+        description: This section contains very important security-related changes.
+        subsections:
+            vuln_fixes:
+                title: Fixed vulnerabilities
+    breaking:
```

Comment:
We should merge #1880 right after this one so that reviewers can see that the 'breaking' section matches the diff to the api/*.txt files.

Reply:
Yes, we just don't need to rush until there are actual breaking changes proposed.

Reply:
Without looking at my draft PRs I can confidently say there will be proposed breaking changes like, straight away :)